### PR TITLE
Us11/merchant item create item

### PR DESCRIPTION
**Pull request by sicknacco**
This PR contains the work for us11 to create a new merchant item
Includes partials for the edit and new pages
Refactored item default status to disabled
New action was added to the merchant items controller
Create action was added to the merchant items controller

### CODE REVIEW

File: app/controllers/merchants/items_controller.rb [hunk 1]
```diff
@@ -30,6 +30,17 @@ def update
...
   end
 
+  def new
+    @merchant = Merchant.find(params[:merchant_id])
+    @item = Item.new
+  end
+  
+  def create
+    merchant = Merchant.find(params[:merchant_id])
+    merchant.items.create(item_params)
```
Comment (thawley2) [inline app/controllers/merchants/items_controller.rb:40]: Future refactor id is to create a flash message for incomplete data. and have a test for sad path.

**Review by abwmn — APPROVED** [review]
nice use of partials

File: app/views/merchants/items/_new_edit.html.erb [hunk 2]
```diff
@@ -0,0 +1,12 @@
+<%= form_with model: model, local: true do |f| %>
```
Comment (thawley2) [inline app/views/merchants/items/_new_edit.html.erb:1]: Great idea for a partial here!

File: spec/features/merchants/items/index_spec.rb [hunk 3]
```diff
@@ -41,9 +41,8 @@
 
     describe 'I see a button next to each item to enable or disable' do
       it 'where the button is enabled for disabled items' do
-        @item2.update(status: 1)
-        @item4.update(status: 1)
-        @item5.update(status: 1)
+        @item1.update(status: 1)
```
Comment (thawley2) [inline spec/features/merchants/items/index_spec.rb:44]: Well our idea of a new item that is added should default to enabled was wrong, at least it was a quick fix.

**Review by thawley2 — APPROVED** [review]
Looks good, put some notes in the for future refactor.

**Pull request merged**